src/Stepper/StepButton.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+// @inheritedComponent ButtonBase
+
+import React, { Children } from 'react';
+import type { Element } from 'react';
+import classNames from 'classnames';
+import withStyles from '../styles/withStyles';
+import ButtonBase from '../ButtonBase';
+import StepLabel from './StepLabel';
+import type { Orientation } from './Stepper';
+
+const isLabel = children => {
+  return children && Children.count(children) === 1 && children.type && children.type === StepLabel;
+};
```

Comment:
We rely on the `muiName` instead of the instance type for the other similar use cases on the codebase. It's allowing more flexibility.

Reply:
Fixed in 3590d4920